### PR TITLE
Change tensorboardX to pytorch.utils.tensorboard

### PR DESCRIPTION
**Pull request by aleksei-kashapov**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def find_version(*file_paths):
                     "graphviz",
                     "jsonschema",
                     "pydot",
-                    "tensorboardX",
+                    "tensorboard",
```
Comment (vshampor) [inline setup.py:53]: Now that we have removed the external `tensorboardX` dependency and no repository code file is using it, why do we introduce another unused dependency?

**Review by vshampor — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (aleksei-kashapov) [reply setup.py:53]: pytorch.utils.tensorboard utilizes tensorboard package

Comment (vshampor) [reply setup.py:53]: Ok, please set a specific version via `==`

**Comment by aleksei-kashapov** [comment]
Jenkins please retry a build

**Review by vshampor — APPROVED** [review]

**Pull request merged**